app/components/split-view.js

```diff
@@ -159,10 +159,19 @@ export default Ember.Component.extend({
   },
 
   minChildPercentage: function(view) {
+    var cssInt = function(name) {
+      return parseInt(view.$().css(name));
+    }
     if(this.get('isVertical')) {
-      return parseInt(view.$().css("min-width")) / this.get('width') * 100 + this.get('sash.widthPercentage') / 2;
+      return (cssInt("min-width") + cssInt("padding-left") + cssInt("padding-right")
```

Comment:
Adding in the margin is needed (I saw a problem it fixed) and border make sense.  I'm going to argue for the moment that since padding is inside the box that it should not be included.